torch/csrc/jit/jit_log.h

```diff
@@ -113,10 +83,10 @@ TORCH_API std::ostream& operator<<(
     std::ostream& out,
     ::torch::jit::JitLoggingLevels level);
 
-#define JIT_LOG(level, ...)                                  \
-  if (is_enabled(__FILE__, level)) {                         \
-    std::cerr << ::torch::jit::jit_log_prefix(               \
-        level, __FILE__, __LINE__, ::c10::str(__VA_ARGS__)); \
+#define JIT_LOG(level, ...)                                          \
+  if (is_enabled(__FILE__, level)) {                                 \
+    get_jit_logging_output_stream() << ::torch::jit::jit_log_prefix( \
```

Comment:
I got some internal testing failures, and I think it's because we're missing "::torch::jit" namespace before "get_jit_logging_output_stream". Do you mind adding it?